bigtable/bigtable_cloudbuild.yaml

```diff
@@ -0,0 +1,6 @@
+steps:
+- name: 'gcr.io/cloud-builders/docker'
```

Comment:
I don't think that this is a good approach to copy/paste these files in all init actions if instead we can create a single script that will execute relevant tests automatically.

Reply:
Are you suggesting using git diff to determine the modified files instead of using the built-in CB functionality that does this for us? If so, the challenges that come to mind are:

- Authenticating whoever is making the changes
- Determining how to git clone a version of the code that contains the changes we want to test. Git cloning from GoogleCloudPlatform/dataproc-initialization-actions would not contain our changes and depending on the user, the repo name would differ.

Reply:
I also thought that having every config copying the code seemed inconvenient, but I assumed that usually not many init action changes are included in a given PR (most commonly only 1?) so the building would only happen once for a given PR.

Reply:
I think we talk past each other, let me describe what I mean with more details:

1. Create a single script that diffs PR changes and runs relevant tests based on them
2. Run this script as entry point of CB integration - I assume that this script will have access  to CB credentials to run tests and will run in git repo context inside CB docker container, so it will have access to git history to diff PR changes and determine what tests it needs to run.

Is this approach not feasible with CB?

Reply:
It should be possible with CB. I'll look more into this to confirm and work on implementing this approach instead.

Reply:
CB now uses a single shell script that runs our tests as the entry point.